### ISSUE TITLE
Plots.jl fails to precompile in Ubuntu 20.10 using official binary v1.5.3

### ISSUE THREAD

**Issue by storopoli** [JuliaLang/julia#39872]
Hey guys, I've download the official binary from Julia (v.1.5.3) and I get this error. I've searched all around in the web, discourse and at the repo's issues and could not find anything. Could you guys help me out? I am using Ubuntu 20.10.

```Julia
julia> using Plots
[ Info: Precompiling Plots [91a5bcdd-55d7-5caf-9e0b-520d859cae80]
ERROR: LoadError: InitError: could not load library "/home/storopoli/.julia/artifacts/7f40eeb66d90d3026ae5fb68761c263b57adb840/lib/libavdevice.so"
/home/storopoli/julia-1.5.3/bin/../lib/julia/libstdc++.so.6: version `GLIBCXX_3.4.25' not found (required by /lib/x86_64-linux-gnu/libopenmpt.so.0)
Stacktrace:
 [1] dlopen(::String, ::UInt32; throw_error::Bool) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Libdl/src/Libdl.jl:109
 [2] dlopen(::String, ::UInt32) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Libdl/src/Libdl.jl:109
 [3] macro expansion at /home/storopoli/.julia/packages/JLLWrappers/WnWcZ/src/products/library_generators.jl:63 [inlined]
 [4] __init__() at /home/storopoli/.julia/packages/FFMPEG_jll/xCSqL/src/wrappers/x86_64-linux-gnu.jl:47
 [5] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:697
 [6] _require_search_from_serialized(::Base.PkgId, ::String) at ./loading.jl:782
 [7] _require(::Base.PkgId) at ./loading.jl:1007
 [8] require(::Base.PkgId) at ./loading.jl:928
 [9] require(::Module, ::Symbol) at ./loading.jl:923
 [10] include(::Function, ::Module, ::String) at ./Base.jl:380
 [11] include(::Module, ::String) at ./Base.jl:368
 [12] top-level scope at none:2
 [13] eval at ./boot.jl:331 [inlined]
 [14] eval(::Expr) at ./client.jl:467
 [15] top-level scope at ./none:3
during initialization of module FFMPEG_jll
in expression starting at /home/storopoli/.julia/packages/FFMPEG/aazvf/src/FFMPEG.jl:3
ERROR: LoadError: Failed to precompile FFMPEG [c87230d0-a227-11e9-1b43-d7ebe4e7570a] to /home/storopoli/.julia/compiled/v1.5/FFMPEG/TGvga_daNcb.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1305
 [3] _require(::Base.PkgId) at ./loading.jl:1030
 [4] require(::Base.PkgId) at ./loading.jl:928
 [5] require(::Module, ::Symbol) at ./loading.jl:923
 [6] include(::Function, ::Module, ::String) at ./Base.jl:380
 [7] include(::Module, ::String) at ./Base.jl:368
 [8] top-level scope at none:2
 [9] eval at ./boot.jl:331 [inlined]
 [10] eval(::Expr) at ./client.jl:467
 [11] top-level scope at ./none:3
in expression starting at /home/storopoli/.julia/packages/Plots/SjqWU/src/Plots.jl:15
ERROR: Failed to precompile Plots [91a5bcdd-55d7-5caf-9e0b-520d859cae80] to /home/storopoli/.julia/compiled/v1.5/Plots/ld3vC_daNcb.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1305
 [3] _require(::Base.PkgId) at ./loading.jl:1030
 [4] require(::Base.PkgId) at ./loading.jl:928
 [5] require(::Module, ::Symbol) at ./loading.jl:923
```

**Comment by Keno** [comment]
You can delete the libstdc++ that ships with julia to have it pick up the system one. However, there seems to be something wrong with the Artifact provided by FFMPEG, since it is picking up the system OpenMP runtime rather one distributed as a JLL. Maybe file an issue on Yggdrasil or FFMPEG.jl for that.

**Comment by storopoli** [comment]
@Keno, Now I get this error on StatsPlots.jl and HTTP.jl. Shall I delete more stuff?

```Julia
(@v1.5) pkg> precompile
Precompiling project...
[ Info: Precompiling StatsPlots [f3b207a7-027a-5e70-b257-86293d7955fd]
ERROR: LoadError: InitError: could not load symbol "fftw_init_threads":
/lib/x86_64-linux-gnu/libfftw3.so.3: undefined symbol: fftw_init_threads
Stacktrace:
 [1] __init__() at /home/storopoli/.julia/packages/FFTW/G3lSO/src/FFTW.jl:48
 [2] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:697
 [3] _require_search_from_serialized(::Base.PkgId, ::String) at ./loading.jl:782
 [4] _tryrequire_from_serialized(::Base.PkgId, ::UInt64, ::Nothing) at ./loading.jl:712
 [5] _require_from_serialized(::String) at ./loading.jl:744
 [6] _require(::Base.PkgId) at ./loading.jl:1040
 [7] require(::Base.PkgId) at ./loading.jl:928
 [8] require(::Module, ::Symbol) at ./loading.jl:923
 [9] include(::Function, ::Module, ::String) at ./Base.jl:380
 [10] include(::Module, ::String) at ./Base.jl:368
 [11] top-level scope at none:2
 [12] eval at ./boot.jl:331 [inlined]
 [13] eval(::Expr) at ./client.jl:467
 [14] top-level scope at ./none:3
during initialization of module FFTW
in expression starting at /home/storopoli/.julia/packages/StatsPlots/y1j3G/src/StatsPlots.jl:24
[ Info: Precompiling HTTP [cd3eb016-35fb-5094-929b-558a96fad6f3]
ERROR: LoadError: InitError: could not load library "/home/storopoli/.julia/artifacts/519367e9365948074c1fcc9f4365597f147a5ab7/lib/libmbedtls.so"
/lib/x86_64-linux-gnu/libmbedx509.so.0: undefined symbol: mbedtls_mutex_unlock
Stacktrace:
 [1] dlopen(::String, ::UInt32; throw_error::Bool) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Libdl/src/Libdl.jl:109
 [2] dlopen(::String, ::UInt32) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Libdl/src/Libdl.jl:109
 [3] macro expansion at /home/storopoli/.julia/packages/JLLWrappers/WnWcZ/src/products/library_generators.jl:63 [inlined]
 [4] __init__() at /home/storopoli/.julia/packages/MbedTLS_jll/qMb7d/src/wrappers/x86_64-linux-gnu.jl:16
 [5] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:697
 [6] _require_search_from_serialized(::Base.PkgId, ::String) at ./loading.jl:782
 [7] _require(::Base.PkgId) at ./loading.jl:1007
 [8] require(::Base.PkgId) at ./loading.jl:928
 [9] require(::Module, ::Symbol) at ./loading.jl:923
 [10] include(::Function, ::Module, ::String) at ./Base.jl:380
 [11] include(::Module, ::String) at ./Base.jl:368
 [12] top-level scope at none:2
 [13] eval at ./boot.jl:331 [inlined]
 [14] eval(::Expr) at ./client.jl:467
 [15] top-level scope at ./none:3
during initialization of module MbedTLS_jll
in expression starting at /home/storopoli/.julia/packages/MbedTLS/4YY6E/src/MbedTLS.jl:3
ERROR: LoadError: LoadError: Failed to precompile MbedTLS [739be429-bea8-5141-9913-cc70e7f3736d] to /home/storopoli/.julia/compiled/v1.5/MbedTLS/bf9T0_daNcb.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1305
 [3] _require(::Base.PkgId) at ./loading.jl:1030
 [4] require(::Base.PkgId) at ./loading.jl:928
 [5] require(::Module, ::Symbol) at ./loading.jl:923
 [6] include(::Function, ::Module, ::String) at ./Base.jl:380
 [7] include at ./Base.jl:368 [inlined]
 [8] include(::String) at /home/storopoli/.julia/packages/HTTP/cxgat/src/HTTP.jl:1
 [9] top-level scope at /home/storopoli/.julia/packages/HTTP/cxgat/src/HTTP.jl:18
 [10] include(::Function, ::Module, ::String) at ./Base.jl:380
 [11] include(::Module, ::String) at ./Base.jl:368
 [12] top-level scope at none:2
 [13] eval at ./boot.jl:331 [inlined]
 [14] eval(::Expr) at ./client.jl:467
 [15] top-level scope at ./none:3
in expression starting at /home/storopoli/.julia/packages/HTTP/cxgat/src/IOExtras.jl:11
in expression starting at /home/storopoli/.julia/packages/HTTP/cxgat/src/HTTP.jl:18
```

**Comment by Keno** [comment]
No, something is messed up where it keeps trying load libraries from your system rather than from the jll.

**Comment by storopoli** [comment]
Do I need to set any `PATH` variables to tell Julia to not do that?

**Comment by storopoli** [comment]
solved in JuliaPackaging/Yggdrasil#2638. It was a `LD_LIBRARY_PATH` thing.
